packages/core/src/actions/contracts/multicall.test.ts

```diff
@@ -50,7 +50,7 @@ describe('multicall', () => {
   })
 
   it('default', async () => {
-    expect(await multicall({ contracts })).toMatchInlineSnapshot(`
+    await expect(multicall({ contracts })).resolves.toMatchInlineSnapshot(`
```

Comment:
not sure why i changed this test file like this, but was into the whole `await expect(…).resolves. toMatchInlineSnapshot(…)` pattern